contracts/ConsensusUtils.sol

```diff
@@ -422,23 +425,27 @@ contract ConsensusUtils is EternalStorage, ValidatorSet {
   function _jailedValidatorRemove(address _address) internal {
     bool found = false;
     uint256 removeIndex;
-    for (uint256 i; i < jailedValidatorsLength(); i++) {
-      if (_address == jailedValidatorsAtPosition(i)) {
-        removeIndex = i;
-        found = true;
-        break;
+    do {
```

Comment:
Hi @Andrew-Pohl
Can you expain why this was changed?

Reply:
Changed it so it removes all occurrences in the list rather than the first

Reply:
Thanks, this is clear now 👍